### ISSUE TITLE
Output doesn't work since commit @750482d

### ISSUE THREAD

**Issue by jeremija** [sardemff7/j4status#20]
The output stopped working completely for me ever since the @750482d commit. I tried it on Arch Linux using glib 2.44.1 and Debian Jessie with glib 2.42.1.

I believe [this line](https://github.com/sardemff7/j4status/blob/master/main/src/io.c#L314) is to blame.

**Comment by sardemff7** [comment]
`header_sent` is set to `TRUE` if `header` is `NULL` so it should work… I tested both `i3bar` and `flat` output.
Which plugins are you using and could you `gdb` or `g_debug` to check the `header_sent` value?

**Comment by jeremija** [comment]
I can see that method is called, but `header_sent` is set to `FALSE` every time. I just ran `j4status` from console with:

```
J4STATUS_PLUGINS_DIR="$PWD/.libs" ./j4status -i time -o flat
```

The only place where `header_sent` is set is from `_j4status_io_stream_put_header` function, it is never called.

**Comment by sardemff7** [comment]
Fixed in 659c14e8d5fc11283c8073bea67f1bcfa483932d.

I just forgot to call `_j4status_io_stream_put_header` for stdout…

**Comment by jeremija** [comment]
Great, thanks!
